pkgs/tools/filesystems/sdat2img/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, lib, fetchFromGitHub, python3, makeWrapper }:
+
+stdenv.mkDerivation {
+  pname = "sdat2img";
+
+  version = "unstable-2021-11-9";
+
+  src = fetchFromGitHub {
+    repo = "sdat2img";
+    owner = "xpirt";
+    rev = "b432c988a412c06ff24d196132e354712fc18929";
+    sha256 = "sha256-NCbf9H0hoJgeDtP6cQY0H280BQqgKXv3ConZ87QixVY=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  buildCommand = ''
+    install -D $src/sdat2img.py $out/bin/sdat2img
+    wrapProgram $out/bin/sdat2img --prefix PATH : ${lib.makeBinPath [ python3 ]}
```

```suggestion
  buildInputs = [ python3 ];

  buildCommand = ''
    install -D $src/sdat2img.py $out/bin/sdat2img
```

The shebang should now be fixed automatically.

---

it does not get fixed automatically sadly :-(
not sure why..

---

along with `format="other"` that worked, thanks!